pkg/daemon/ceph/agent/cluster/controller.go

```diff
@@ -66,7 +66,7 @@ func (c *ClusterController) StartWatch(namespace string, stopCh chan struct{}) {
 	}
 
 	logger.Infof("start watching cluster resources")
-	go k8sutil.WatchCR(opcontroller.ClusterResource, namespace, resourceHandlerFuncs, c.context.RookClientset.CephV1().RESTClient(), &cephv1.CephCluster{}, stopCh)
+	go k8sutil.WatchCR(opcontroller.ClusterResource, namespace, resourceHandlerFuncs, c.context.RookClientset.CephV1().RESTClient(), &cephv1.CephCluster{}, stopCh) //nolint
```

Comment:
Wherever nolint is used, how about a brief description?

Reply:
Done.